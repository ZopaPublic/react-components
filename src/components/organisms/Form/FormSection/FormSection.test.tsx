import React from 'react';
import { render } from '@testing-library/react';
import { FormSection } from '..';

describe('<FormSection />', () => {
  it('renders the form section', () => {
    const container = render(
      <FormSection title="Form Section" subtitle="Subtitle">
        <div>Child 1</div>
        <div>Child 2</div>
      </FormSection>,
    );
    expect(container).toMatchSnapshot();
  });
});
