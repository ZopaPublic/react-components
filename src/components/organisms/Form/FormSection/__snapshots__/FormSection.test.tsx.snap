// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormSection /> renders the form section 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c1 {
  margin: 0;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  color: #2C3236;
  font-size: 16px;
  line-height: 24px;
  font-weight: 400;
  font-family: "Open Sans",Roboto,Helvetica,Arial,sans-serif;
  text-align: inherit;
  display: block;
}

.c0 {
  font-size: 18px;
  line-height: 26px;
  -webkit-letter-spacing: -0.02px;
  -moz-letter-spacing: -0.02px;
  -ms-letter-spacing: -0.02px;
  letter-spacing: -0.02px;
  margin: 0;
  color: #2C3236;
  text-align: inherit;
  font-family: "Open Sans",Roboto,Helvetica,Arial,sans-serif;
  font-weight: 700;
}

<body>
    <div>
      <div
        class="py-6"
      >
        <h5
          class="c0 mb-2"
        >
          Form Section
        </h5>
        <span
          class="c1 mb-6"
        >
          Subtitle
        </span>
        <div
          class="mb-4"
        >
          <div>
            Child 1
          </div>
        </div>
        <div>
          <div>
            Child 2
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="py-6"
    >
      <h5
        class="sc-jDwBTQ gvnpFQ mb-2"
      >
        Form Section
      </h5>
      <span
        class="sc-ifAKCX llEtYb mb-6"
      >
        Subtitle
      </span>
      <div
        class="mb-4"
      >
        <div>
          Child 1
        </div>
      </div>
      <div>
        <div>
          Child 2
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
